pkg/locale/localizer.go

```diff
@@ -60,23 +110,31 @@ const (
 
 // domain events
 const (
-	UnknownOperation   = "UnknownOperation"
-	CreatedTemplate    = "Created"
-	EnabledTemplate    = "Enabled"
-	DisabledTemplate   = "Disabled"
-	ArchivedTemplate   = "Archived"
-	UnarchivedTemplate = "Unarchived"
-	DeletedTemplate    = "Deleted"
-	AddedTemplate      = "Added"
-	ChangedTemplate    = "Changed"
-	UpdatedTemplate    = "Updated"
-	StartedTemplate    = "Started"
-	StoppedTemplate    = "Stopped"
-	FinishedTemplate   = "Finished"
-	UploadedTemplate   = "Uploaded"
-	ResetTemplate      = "Reset"
-	ClonedTemplate     = "Cloned"
-	TrialConverted     = "TrialConverted"
+	UnknownOperation           = "UnknownOperation"
+	CreatedTemplate            = "Created"
+	EnabledTemplate            = "Enabled"
+	DisabledTemplate           = "Disabled"
+	ArchivedTemplate           = "Archived"
+	UnarchivedTemplate         = "Unarchived"
+	DeletedTemplate            = "Deleted"
+	AddedTemplate              = "Added"
+	ConditionAddedTemplate     = "ConditionAdded"
+	ConditionDeletedTemplate   = "ConditionDeleted"
+	ChangedTemplate            = "Changed"
+	DescriptionUpdatedTemplate = "DescriptionUpdated"
+	IncrementedTemplate        = "Incremented"
+	NameUpdatedTemplate        = "NameUpdated"
+	UpdatedTemplate            = "Updated"
+	StartedTemplate            = "Started"
+	StoppedTemplate            = "Stopped"
+	FinishedTemplate           = "Finished"
+	UploadedTemplate           = "Uploaded"
+	ResetTemplate              = "Reset"
+	ClonedTemplate             = "Cloned"
+	TrialConverted             = "TrialConverted"
```

Comment:
Is the variable name `TrialConvertedTemplate`?
Or not for a template?

Reply:
I think when it was created, he didn't use the template suffix because it doesn't receive a field.
Same as for the `UnknownOperation`.